context/app/static/js/components/workspaces/CreateWorkspaceDialog/CreateWorkspaceDialog.jsx

```diff
@@ -36,11 +37,22 @@ function CreateWorkspaceDialog({
             id="create-workspace-form"
             component="form"
             sx={{
+              display: 'grid',
+              gap: 2,
               marginTop: 1,
             }}
             onSubmit={handleSubmit(onSubmit)}
           >
-            <CreateWorkspaceInput control={control} name="name" errors={errors} />
+            {protectedHubmapIds.length > 0 && (
+              <WorkspaceField
+                control={control}
+                name="Protected Datasets"
+                errors={errors}
+                // eslint-disable-next-line no-underscore-dangle
```

Comment:
This eslint ignore can be dropped since the line with `_source` in it is now in `MetadataMenu`